src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -275,4 +280,19 @@ private function addFlashMessageProductV2IsDisabled(): void
             )
         );
     }
+
+    private function addFlashMessageNotMultistoreCompatible(): void
```

Comment:
Do we need a method for this as it's used just one time?
(If you add the message for edit action, you probably will reuse it)

Reply:
I find it more readable, and same was done for `addFlashMessageProductV2IsDisabled` previously, I wanted to keep this pattern

Reply:
(and yes I will actually use it in the edition controller method)

Reply:
I'm removing this method anyway :D so I resolve this feedback